### ISSUE TITLE
OpenScad Location

### ISSUE THREAD

**Issue by 5axes** [kartchnb/AutoTowersGenerator#4]
Would be nice to add an option to precify the location of OpenScad.

In My case it's not in : PROGRAMFILES(X86)  nor PROGRAMFILES   

Using a special location

**Comment by kartchnb** [comment]
That's a great suggestion and something I tried implementing.  I wasn't able to get string preferences to save between sessions.  Numeric preferences were fine, but strings didn't seem to work right.  

It's been a while since I looked at it, and it's probably just due to me misunderstanding how the Uranium/Cura interface works.  I'll give it another shot.

Thanks for looking at this.  I was hugely inspired by your calibration shapes plugin, which I use all the time.
